docs/sso-integrations/integrate-with-okta.md

```diff
@@ -3,27 +3,26 @@ title: Integrate With Okta
 sidebar_position: 2
 ---
 
-**[ROUGH DRAFT, needs review for accuracy and editing]**
+This guide describes how to configure Beyond Identity as an Identity Provider for Okta using the OpenID Connect identity protocol. 
+The steps are: 
 
+## Setup a Beyond Identity tenant
 
-This guide describes how to configure Beyond Identity with an use an OIDC authentication flow without a management API resource server. 
-This flow should be used if a client is using Beyond Identity for authentication only. 
+[Create a tenant](https://www.beyondidentity.com/developers/signup) via the sign up form.
 
-## Setup the Beyond Identity tenant
+After a tenant and a local Passkey is created, you will be automatically redirected to the [Admin Console](http://console-us.beyondidentity.run/).
 
-Create a tenant via the self signup form:
+### Create a new realm
```

Comment:
Should we add a little blurb about why we're requiring them to create a new realm?

Reply:
perfect!